### PR TITLE
Add pci label COPS-357

### PR DESCRIPTION
**Pull request by ankitbhatia32**
This PR adds the workflow to run the Labeler github action on pull requests as well as the pci label.

After merging this PR all new pull requests that modify files under datadog-agent will require at least 2 reviews (author plus one reviewer) to merge.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by mx-psi — DISMISSED** [review]
You need to sign/fix the CLA

**Review by mx-psi — COMMENTED** [review]
If all changes on the datadog-agent repo are labeled 'pci', wouldn't it be easier to just change this at the repo-configuration level?

**Comment by ankitbhatia32** [comment]
Hi @julien-lebot , my apologies for the delay in response. We were waiting on approval from our GRC team to add these labels and we just got the approval this week. Can you please review and merge this PR now?

**Comment by mx-psi** [comment]
@ankitbhatia32 The PR LGTM, but I would love to get some context as to why we are doing it this way instead of changing the configuration at the repo level

**Pull request closed without merging**
